ai2thor/video_controller.py

```diff
@@ -287,9 +262,14 @@ def AbsoluteCameraAnimation(self,
                                  'y': p0['y'] + (py - p0['y']) / frames * i,
                                  'z': p0['z'] + (pz - p0['z']) / frames * i},
                 }
-                if not (smartSkybox and maxY > p0['y'] + (py - p0['y']) / frames * i):
-                    kwargs['skyboxColor'] = 'black'
-                yield self.step(**kwargs)
+
+            # enables linear animation changes to the camera FOV
+            if FOVstart != None and FOVend != None:
+                kwargs['fieldOfView'] = FOVstart + (FOVend - FOVstart) / frames * i
```

Comment:
Don't you have to call `reset` and `TeleportFull` here too?

Reply:
i do not believe so, this was getting called because it was a new camera, now it is an updated camera
@mattdeitke can confirm

Reply:
The video controller updates look good to me! (Assuming updating the ThirdPartyCamera's FOV on the Unity side works as expected.)